app/assets/javascripts/misc/previous-address.js

```diff
@@ -0,0 +1,37 @@
+import 'classlist.js';
+
+const I18n = window.LoginGov.I18n;
+
+function previousAddress() {
+  const accordion = document.querySelector('.accordion');
+
+  if (accordion) {
+    const controls = accordion.querySelector('[aria-controls]');
+    const selects = accordion.querySelectorAll('select');
+    const inputs = accordion.querySelectorAll('input');
+
+    controls.classList.remove('display-none');
```

Comment:
I think I'm missing something here, why do we add a `display-none` class to the header in the html, and then remove it as soon as the `DOMContentLoaded` event fires?